subprojects/docs/docs.gradle

```diff
@@ -469,6 +469,17 @@ samples {
                 from(sampleDirectory.dir("common"))
             }
         }
+
+        publishingCredentials {
+            sampleDirectory = samplesRoot.dir("user-input/publishing-credentials")
+            description = "Provide credentials to a publishing task."
+            category = "User input"
```

Comment:
Category should probably change to `Supplying credentials` together with the one in https://github.com/gradle/gradle/pull/13006
Kept it the same for now.